From: Sophie Brun <sophie@offensive-security.com>
Date: Wed, 8 Jul 2020 11:53:53 +0200
Subject: Change sqlite dir to grant user access

---
 theHarvester/lib/stash.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/theHarvester/lib/stash.py b/theHarvester/lib/stash.py
index c5ae21a..ca3f152 100644
--- a/theHarvester/lib/stash.py
+++ b/theHarvester/lib/stash.py
@@ -1,11 +1,15 @@
 import aiosqlite
 import datetime
+import os
 
+db_path = os.path.expanduser("~/.local/share/theHarvester")
+if not os.path.isdir(db_path):
+    os.makedirs(db_path)
 
 class StashManager:
 
     def __init__(self):
-        self.db = "stash.sqlite"
+        self.db = os.path.join(db_path, "stash.sqlite")
         self.results = ""
         self.totalresults = ""
         self.latestscandomain = {}
